frontend/client/src/styles/_loading.scss

```diff
@@ -34,6 +34,20 @@ $loading-offset: 25px;
 		width: 90%;
 		max-width: 1200px;
 	}
+
+	@media (max-width: 800px) {
```

Comment:
Could you restructure these media queries to be nested within the `[data-letter-crap]` styles above? It should look something like the following:

```css
[data-letter-crap] {
	position: relative;
	white-space: pre;
	overflow: hidden;
	font-family: monospace;
	padding: 0;
	color: $green;
	width: 90%;
	max-width: 1200px;
	font-size: 10px;
	line-height: 10px;

  @media (max-width: 800px) {
		font-size: 6.5px;
		line-height: 6.5px;
	}

	...
```

Thanks!